staging/src/k8s.io/api/batch/v1/types.go

```diff
@@ -336,6 +349,11 @@ type JobSpec struct {
 	//
 	// +optional
 	Suspend *bool `json:"suspend,omitempty" protobuf:"varint,10,opt,name=suspend"`
+
+	// Only allow pods to be started once the terminating pods have been
```

Comment:
Explain the field, the possible values and the default. You can see in the swagger.json file that only this comment makes it into it.

Also explain that this is alpha and needs the feature gate to be enabled.

Reply:
Thanks for calling that out.  I will use the comments in the KEP for this API field and then we can iterate on that.